RunForrest/dist/index.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+
+<html>
+<head>
+<meta http-equiv="Content-Type" content="text/html"; charset="UTF-8" />    
+<title>Diseño Web</title>
+<link href="main.css" rel="stylesheet" type="text/css">    
+<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+<script src="https://cdnjs.cloudflare.com/ajax/libs/mustache.js/2.1.0/mustache.min.js"></script>
+<script type="text/javascript" src="js/main.js"></script>
```

Comment:
Include external scripts at the bottom of the body section. 
https://faqs.skillcrush.com/article/176-where-should-js-script-tags-be-linked-in-html-documents